e2etests/test/c2d_disconnect.js

```diff
@@ -87,7 +88,7 @@ var protocolAndTermination = [
     delayInSeconds: 2
   },
   {
-    testEnabled: true,
+    testEnabled: false,
```

Comment:
>false, [](start = 17, length = 6)

?

Reply:
fixed in iteration 3

---
In reply to: [146330356](https://github.com/Azure/azure-iot-sdk-node/pull/114#discussion_r146330356) [](ancestors = 146330356)